<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCameraControlsDialogClass</class>
 <widget class="QDialog" name="QtCameraControlsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCameraControlsDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Camera Device: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCameraDevice">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupFormat">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <widget class="QPushButton" name="buttonResetFormat">
        <property name="text">
         <string>Reset Format</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>FPS</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dropdownPixelFormat"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="dropdownFps"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="buttonSeeAllFormats">
        <property name="text">
         <string>See All Formats</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Pixel Format</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="dropdownResolution"/>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioFps">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">radioGroupFormat</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioResolution">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">radioGroupFormat</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="radioPixelFormat">
        <property name="text">
         <string/>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">radioGroupFormat</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="horizontalSpacing">
       <number>9</number>
      </property>
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Active</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkboxActive">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color Temperature</string>
        </property>
        <property name="buddy">
         <cstring>sliderColorTemp</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="sliderColorTemp">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Exposure Mode</string>
        </property>
        <property name="buddy">
         <cstring>dropdownExposureMode</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dropdownExposureMode">
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manual</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Portrait</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Night</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sports</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Snow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Beach</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Landscape</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Night Portrait</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Theatre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sunset</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Steady Photo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fireworks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Party</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Candlelight</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Barcode</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Exposure Comp.</string>
        </property>
        <property name="buddy">
         <cstring>sliderExposureComp</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="sliderExposureComp">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Flash Mode</string>
        </property>
        <property name="buddy">
         <cstring>dropdownFlashMode</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="dropdownFlashMode">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>On</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Auto Exposure Time</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="checkboxAutoExposureTime">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Manual Exposure Time</string>
        </property>
        <property name="buddy">
         <cstring>sliderManualExposureTime</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSlider" name="sliderManualExposureTime">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Auto Iso Sensitivity</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="checkboxAutoIsoSensitivity">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Manual ISO Sensitivity</string>
        </property>
        <property name="buddy">
         <cstring>sliderManualIsoSensitivity</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSlider" name="sliderManualIsoSensitivity">
        <property name="maximum">
         <number>128000</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Torch Mode</string>
        </property>
        <property name="buddy">
         <cstring>dropdownTorchMode</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="dropdownTorchMode">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>On</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>White Balance Mode</string>
        </property>
        <property name="buddy">
         <cstring>dropdownWhiteBalanceMode</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="dropdownWhiteBalanceMode">
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manual</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sunlight</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cloudy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shade</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tungsten</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fluorescent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Flash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sunset</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::RestoreDefaults|QDialogButtonBox::StandardButton::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radioGroupFormat"/>
 </buttongroups>
</ui>
